packages/console/src/pages/Connectors/SocialConnectors.tsx

```diff
@@ -0,0 +1,111 @@
+import { ConnectorDTO, ConnectorType } from '@logto/schemas';
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import useSWR from 'swr';
+
+import placeholder from '@/assets/images/social-connectors-placeholder.svg';
+import Button from '@/components/Button';
+import Card from '@/components/Card';
+import CardTitle from '@/components/CardTitle';
+import ImagePlaceholder from '@/components/ImagePlaceholder';
+import ItemPreview from '@/components/ItemPreview';
+import Status from '@/components/Status';
+import TabNav from '@/components/TabNav';
+import TabNavLink from '@/components/TabNav/TabNavLink';
+import { RequestError } from '@/swr';
+
+import * as styles from './index.module.scss';
+
+const SocialConnectors = () => {
+  const { t, i18n } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const { data, error } = useSWR<ConnectorDTO[], RequestError>('/api/connectors');
+  const isLoading = !data && !error;
+
+  const socialConnectors = useMemo(() => {
+    if (!data) {
+      return null;
+    }
+
+    return data.filter((connector) => connector.metadata.type === ConnectorType.Social);
+  }, [data]);
+
+  return (
+    <Card>
+      <div className={styles.headline}>
+        <CardTitle title="connectors.title" subtitle="connectors.subtitle" />
+        <Button disabled title="admin_console.connectors.create" />
+      </div>
+      <TabNav>
+        <TabNavLink href="/connectors">{t('connectors.tab_email_sms')}</TabNavLink>
+        <TabNavLink href="/connectors/social">{t('connectors.tab_social')}</TabNavLink>
+      </TabNav>
+      <table className={styles.table}>
+        <thead>
+          <tr>
+            <td className={styles.connectorName}>{t('connectors.connector_name')}</td>
+            <td>{t('connectors.connector_type')}</td>
+            <td>{t('connectors.connector_status')}</td>
+          </tr>
+        </thead>
+        <tbody>
+          {error && (
+            <tr>
+              <td colSpan={3}>error occurred: {error.metadata.code}</td>
+            </tr>
+          )}
+          {isLoading && (
+            <tr>
+              <td colSpan={3}>loading</td>
```

Comment:
Text content should be loaded from i18n templates

Reply:
This is just a placeholder, we will have "Loading Spinner" design.